src/Verify.MSTest/TestExecutionContext.cs

```diff
@@ -0,0 +1,30 @@
+namespace VerifyMSTest;
+
+public record class TestExecutionContext
+{
+    public TestContext TestContext { get; }
+    public Assembly TestAssembly { get; }
+    public Type TestClass { get; }
+    public MethodInfo TestMethod { get; }
+
+    public TestExecutionContext(TestContext testContext, Type classType)
```

Comment:
@SimonCropp open to suggestions for a name here. Honestly "TestContext" fits well, but the ambiguous type name makes things somewhat complicated.

Reply:
happy with your choice